pkg/auth/auth.go

```diff
@@ -82,6 +93,25 @@ func PersistConfluentLoginToConfig(cfg *config.Config, credentials *Credentials,
 	return addOrUpdateContext(cfg, false, credentials, ctxName, url, state, caCertPath, "", save)
 }
 
+func getSsoSub(authToken string) (string, error) {
```

Comment:
We may want to use this for non-SSO logins in the future (mostly just to simplify the code)
```suggestion
func getSub(authToken string) (string, error) {
```